app/scripts/views/mixins/service-mixin.js

```diff
@@ -12,6 +12,10 @@ define(function (require, exports, module) {
   const $ = require('jquery');
 
   module.exports = {
+    afterRender () {
+      this.transformLinks();
```

Comment:
TBH, there may be, though I imagine it's pretty small since `transformLinks` only looks in the view's DOM tree for matching elements, not the entire DOM.